cypress/e2e/circulation-log/claimed-returned.cy.js

```diff
@@ -0,0 +1,104 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import permissions from '../../support/dictionary/permissions';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../support/fragments/users/usersCard';
+import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+import devTeams from '../../support/dictionary/devTeams';
+import Users from '../../support/fragments/users/users';
+import UserEdit from '../../support/fragments/users/userEdit';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Checkout from '../../support/fragments/checkout/checkout';
+import LoansPage from '../../support/fragments/loans/loansPage';
+import ItemsOperations from '../../support/fragments/inventory/inventoryItem/itemsOperations';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+
+let userId;
+let source;
+let servicePointId;
+const item = {
+  instanceName: `Barcode search test ${Number(new Date())}`,
+  ITEM_BARCODE: `123${getRandomPostfix()}`,
+};
+
+describe('circulation-log', () => {
+  before('create checked out item', () => {
+    cy.createTempUser([
+      permissions.inventoryAll.gui,
+      permissions.circulationLogAll.gui,
+      permissions.usersViewRequests.gui
+    ])
+      .then(userProperties => {
+        userId = userProperties.userId;
+        cy.login(userProperties.username, userProperties.password, { path: TopMenu.circulationLogPath, waiter: SearchPane.waitLoading });
+        cy.getAdminToken()
+          .then(() => {
+            cy.getLoanTypes({ limit: 1 });
+            cy.getMaterialTypes({ limit: 1 });
+            cy.getLocations({ limit: 1 });
+            cy.getHoldingTypes({ limit: 1 });
+            source = InventoryHoldings.getHoldingSources({ limit: 1 });
+            cy.getInstanceTypes({ limit: 1 });
+            ServicePoints.getViaApi({ limit: 1, query: 'pickupLocation=="true"' })
+              .then((res) => {
+                servicePointId = res[0].id;
+              });
+            cy.getUsers({
+              limit: 1,
+              query: `"personal.lastName"="${userProperties.username}" and "active"="true"`
+            });
+          })
+          .then(() => {
+            UserEdit.addServicePointViaApi(servicePointId, userId);
+            cy.getUserServicePoints(Cypress.env('users')[0].id);
+            InventoryInstances.createInstanceViaApi(item.instanceName, item.ITEM_BARCODE);
+          })
+          .then(() => {
+            Checkout.checkoutItemViaApi({
+              itemBarcode: item.ITEM_BARCODE,
+              userBarcode: Cypress.env('users')[0].barcode,
+              servicePointId,
+            });
+          });
+        cy.loginAsAdmin();
+        cy.visit(TopMenu.usersPath);
+        // cy.loginAsAdmin({ path: TopMenu.usersPath, waiter: UsersSearchPane.waitLoading })
```

Comment:
remove comment pls
I agree to leave loginAsAdmin and visit as separation function here

Reply:
done